src/plugins/intel_gpu/src/graph/impls/onednn/reorder_onednn.cpp

```diff
@@ -32,12 +32,14 @@ struct reorder_onednn : typed_primitive_onednn_impl<reorder, dnnl::reorder::prim
         int input_idx = DNNL_ARG_FROM;
         for (size_t i = 0; i < instance.inputs_memory_count(); i++) {
             auto& input = instance.input_memory(i);
-            args.insert({input_idx++, input.get_onednn_memory(_pd.src_desc())});
+            auto offset = onednn::get_f_offset(instance.get_input_layout(i), _pd.dnnl::primitive_desc_base::src_desc(0));
```

Comment:
Is it correct to use 0 in `_pd.dnnl::primitive_desc_base::src_desc(0)`? I think it should be `i`...

Reply:
Yes, correct. I'll update it